### PR TITLE
delete outdated options for nucscen, keep only number 2 (default), 5 and 6 

### PR DESCRIPTION
**Pull request by LaviniaBaumstark**
# Purpose of this PR
cleaning options of nucscen

## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [x] Refactoring

## Checklist:

- [x] My code follows the coding etiquette
- [x] I have performed a self-review of my own code
- [x] I have updated the in-code documentation

## Further information (optional):

* Test runs are here: 
* Comparison of results (what changes by this PR?):

### CODE REVIEW

File: modules/40_techpol/NDC/bounds.gms [hunk 1]
```diff
@@ -31,8 +26,6 @@ if(cm_nucscen eq 5,
   vm_cap.lo(t,regi_nucscen,"tnrs","1")$(t.val gt 2020) = 0;
 );
```
Comment (mikapfl) [inline modules/40_techpol/NDC/bounds.gms:27]: why do we have this only in NDC and not in the other realizations of 40_techpol? And isn't this redundant with the bound in bounds.gms which applies starting 2015?
https://github.com/LaviniaBaumstark/remind/blob/69cb5f61eafc3a5bab0f25dfe5e4afb27934a7af/core/bounds.gms#L222

Comment (LaviniaBaumstark) [reply modules/40_techpol/NDC/bounds.gms:27]: I assume that this is specific for some national plans and the other realizations are not as up to date? 
In core/bounds.gms it is only active for nucscen=5

Comment (mikapfl) [reply modules/40_techpol/NDC/bounds.gms:27]: here in NDC/bounds it is also only active for nucscen=5, right?

Comment (LaviniaBaumstark) [reply modules/40_techpol/NDC/bounds.gms:27]: hm, right, I will also delete this part...

**Review by mikapfl — APPROVED** [review]

**Pull request merged**